src/bundle/ui-dev/src/modules/universal-discovery/universal.discovery.module.js

```diff
@@ -137,7 +137,23 @@ export const SearchTextContext = createContext();
 export const DropdownPortalRefContext = createContext();
 
 const UniversalDiscoveryModule = (props) => {
-    const { tabs } = ibexa.adminUiConfig.universalDiscoveryWidget;
+    const { tabs: tabsWithPriority } = ibexa.adminUiConfig.universalDiscoveryWidget;
+    const tabs = tabsWithPriority.reduce((tabsPrioritized, tabToAdd) => {
```

Comment:
maybe put it in memo? Not sure if it's easy as you have to pass array

Reply:
I removed this approach.